hazelcast-jet-core/src/main/java/com/hazelcast/jet/Sources.java

```diff
@@ -176,6 +187,8 @@ private Sources() {
      *      ADDED} and {@link com.hazelcast.core.EntryEventType#UPDATED UPDATED} events
      * @param projectionFn the projection to map the events, you may use
      *     {@link Util#mapEventToEntry()} to project new value from the event
+     *     If the projection returns a {@code null} for some items, that items
```

Comment:
Should be "If the projection returns {@code null} for an item, that item will be filtered out. There are some more instances of this text below.